### PR TITLE
Added Odin Project

### LINKED ISSUE #1623: [Add] <The Odin Project to Frontend category>
### Site URL

https://www.theodinproject.com/

### Description of the above provided site

It is a free, open-source online coding curriculum. It offers courses covering web development basics, JavaScript, and Ruby on Rails. The curriculum is supported by an open-source community and maintains a Discord community of nearly 718,000 learners.

### Category

Frontend

### Sub Category

fonts

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)

### PR DESCRIPTION
**Pull request by mrigank590**
## Fixes Issue

Closes #1623

## Changes proposed

Added Odin Project link

## Screenshots

image

## Note to reviewers

In the issue I had selected the wrong subcategory "fonts" while I should have selected "React Js"

### CODE REVIEW

File: database/frontend/react.json [hunk 1]
```diff
@@ -107,5 +107,12 @@
     "category": "frontend",
     "subcategory": "react",
     "language": "hindi"
+  },
+  {
+    "name": "Odin Project",
+    "description": "It is a free, open-source online coding curriculum. It offers courses covering web development basics, JavaScript, and Ruby on Rails. The curriculum is supported by an open-source community and maintains a Discord community of nearly 718,000 learners.",
```
Comment (CBID2) [inline database/frontend/react.json:113]:
```suggestion
    "description": "The Odin Project is a free, online coding curriculum maintained by the open source community. It offers courses covering web development basics, JavaScript,Ruby on Rails, job search strategies. It also has a Discord community of nearly 718,000 learners.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Made a few suggestions on how to improve it @mrigank590

**Review by CBID2 — APPROVED** [review]
LGTM

**Review by Anmol-Baranwal — COMMENTED** [review]
@rupali-codes 

Please give your opinion regarding the category since this includes both the backend and frontend topics.

image

**Comment by mrigank590** [comment]
@Anmol-Baranwal this is the same case with Fullstack Open so i was thinking of making a new category of fullstack, with which i am almost done but how will i make changes to this pull request for that?
i haven't used github much so yeah

**Comment by Anmol-Baranwal** [comment]
> @Anmol-Baranwal this is the same case with Fullstack Open so i was thinking of making a new category of fullstack, with which i am almost done but how will i make changes to this pull request for that? i haven't used github much so yeah

Don't worry. We will not ask you to make a new category. Also, if you want to create one, just tag any of the moderators. And we will help you out with anything, whether it's github or code.

**Comment by mrigank590** [comment]
> > @Anmol-Baranwal this is the same case with Fullstack Open so i was thinking of making a new category of fullstack, with which i am almost done but how will i make changes to this pull request for that? i haven't used github much so yeah
> 
> Don't worry. We will not ask you to make a new category. Also, if you want to create one, just tag any of the moderators. And we will help you out with anything, whether it's github or code.

actually i am almost done with making a new category and i will put a new issue tomorrow 
and one more stupid question where i can check who the moderators are?

**Comment by Anmol-Baranwal** [comment]
> > > @Anmol-Baranwal this is the same case with Fullstack Open so i was thinking of making a new category of fullstack, with which i am almost done but how will i make changes to this pull request for that? i haven't used github much so yeah
> > 
> > 
> > Don't worry. We will not ask you to make a new category. Also, if you want to create one, just tag any of the moderators. And we will help you out with anything, whether it's github or code.
> 
> actually i am almost done with making a new category and i will put a new issue tomorrow and one more stupid question, where can i check who the moderators are?

I knew you would ask this.

It's not listed, which we will do for sure.

You can tag the moderators by their names: just use `@name` and you will get the option

> `rupali-codes` `k-deepak04` `CBID2` `Anmol-Baranwal`

**Comment by rupali-codes** [comment]
> @rupali-codes
> 
> Please give your opinion regarding the category since this includes both the backend and frontend topics.
> 
> image

its **database**

**Review by Gdakgdak22 — APPROVED** [review]

**Review by rupali-codes — APPROVED** [review]
lgtm

**Comment by Anmol-Baranwal** [comment]
@rupali-codes 
He didn't change the subcategory, like this is still in react.

> I'm just confirming.

**Pull request merged**